### DISCUSSION TITLE
Different settings for Elevated shells?

### DISCUSSION THREAD

**Discussion by LispEngineer** [microsoft/terminal#12178] [Q&A] [answered]
Hi there,

I was wondering if there was a way to have different settings apply when running Windows Terminal as administrator (elevated privileges) or when one tab becomes elevated (e.g., just "gsudo" in PowerShell). The Terminal application knows when this happens, as it shows "Administrator: PowerShell (winget)" (on mine) instead of just "PowerShell (winget)" in the tab title.

My use case would be to make the background red, or the title bar red, or something that makes it immistakably clear when I'm in an elevated session.

Thank you to the team which brought this tool to life - it has made my transition to Windows immeasurably better.

Thanks!

**Comment by zadjii-msft** [comment, marked as answer]
Right now we don't have a good way of doing that. 

In 1.13 you'll be able to mark a profile as requiring elevation (and it'll auto elevate itself), but there's no good way to change the settings of a profile depending on the elevation status.

You'll want to follow #8311, which we're using to track adding support for exactly this 😉
